.github/workflows/i18n.yaml

```
@@ -5,8 +5,8 @@ concurrency:
 on:
   workflow_dispatch:
   schedule:
-    # Daily at 2:00 a.m.
-    - cron: '0 2 */3 * *'
+    # Monday and Thursday at 2:00 a.m.
+    - cron: '0 2 * * 1,4'
```

I'd prefer to have the translations update running more often than only twice a week to get translation changes into the game for testing faster, especially considering #5549 
```suggestion
    # Monday, Thursday, Saturday at 2:00 a.m.
    - cron: '0 2 * * 1,4,6'
```